src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js

```diff
@@ -114,16 +113,14 @@
         }
 
         const contentContainer = lastSection.closest('.ibexa-edit-content__container');
-
-        contentContainer.style.paddingBottom = '0px';
+        const sectionsMainContainer = lastSection.closest('.ibexa-edit-content');
 
         if (!firstSection.isSameNode(lastSection) && lastSection.offsetHeight) {
-            const heightFromLastSection = contentContainer.offsetHeight + contentContainer.offsetTop - lastSection.offsetTop;
-            const contentColumnBodyHeight = contentColumn.offsetHeight - headerContainer.offsetHeight;
-            const heightDiff = contentColumnBodyHeight - heightFromLastSection;
+            const lastSectionHeight = lastSection.offsetHeight;
```

Comment:
You can't remove previous logic, if you add sth to group `content-edit-sections` or `content-edit-form-after` its broken then.
You can easily test it by adding anything before this line https://github.com/ibexa/admin-ui/blob/3fefb3ef339e6f45d6ef8a64249a1eecd733a39e/src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig#L72
or this line
https://github.com/ibexa/admin-ui/blob/3fefb3ef339e6f45d6ef8a64249a1eecd733a39e/src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig#L83

This is how it should behave (current 4.2 branch):
![Selection_508](https://user-images.githubusercontent.com/4689171/195807296-b55f1d3a-4976-41cc-a627-2d126f339546.png)

This is how it behave in your PR:
https://user-images.githubusercontent.com/4689171/195807863-f30bc8f7-1a44-4e13-9455-957dc546e11c.mp4